CMSIS/NN/Tests/UnitTest/TestCases/TestData/avgpooling/weights_data.h

```diff
@@ -0,0 +1,94 @@
+
+/*
+ * Copyright (C) 2010-2020 Arm Limited or its affiliates. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+// Generated by generate_test_data.py
+#include <stdint.h>
+
+const q7_t avgpooling_weights[68] =
```

Comment:
Please change this to int8_t

Reply:
Ok

Reply:
Actually it was a mistake to add it. Removing it.